app/controllers/appointment_summaries_controller.rb

```diff
@@ -20,14 +18,7 @@ def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
 
-    respond_to do |format|
-      format.html { render html: output_document.html.html_safe }
-      format.pdf do
-        send_data output_document.pdf,
-                  filename: 'pension_wise.pdf', type: 'application/pdf',
-                  disposition: :inline
-      end
-    end
+    render html: output_document.html.html_safe
```

Comment:
Do we want to render a preview of the actual document or just a confirmation view?

If we do we should defiantly extract the template into a shareable bower module.